pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -873,6 +873,9 @@ type Parameter struct {
 
 	// Description is the parameter description
 	Description *AnyString `json:"description,omitempty" protobuf:"bytes,7,opt,name=description"`
+
+	// Description is the parameter description
```

Comment:
```suggestion
	// Whether to allow multi-select in the UI
```

this was previously a copy+paste of the above `Description` field, so this comment doesn't add up. Per the codegen, it also appears in many other places as the API description of this field, so the comment is important

Reply:
Sorry for that, in the beginning I did change the description of this field, though after that I ran codegen and recreated the commits according to standards and apparently something was deleted, I'll fix that.